src/views/MetadataView/Meditor.vue

```diff
@@ -309,6 +299,17 @@ export default defineComponent({
 
     const TransactionTracker = new MeditorTransactionTracker(editorInterface);
 
+    const closeEditor = () => {
```

Comment:
This looks like it closes without prompting if the user has unsaved work. Am I missing something?

Reply:
`closeEditor()` is called after the check for unsaved work happens. 
First `exitMeditor()` is called, which checks whether or not there is unsaved work. If there isn't, it calls `closeEditor()`. If there is, the unsaved changes popup comes up, and clicking "yes" on that will call `closeEditor()`.